lib/commands/validate-example.ts

```diff
@@ -30,8 +30,8 @@ export function handler(argv: yargs.Arguments) {
     logFilepath: argv.f,
   }
   if (specPath.match(/.*composite.*/ig) !== null) {
```

Comment:
> if (specPath.match(/.*composite.*/ig) !== null) { [](start = 1, length = 50)

maybe regex.test(string) instead if we only care about if it matches ? not sure how much of a concern is here, but test is faster

Reply:
same in other places

---
In reply to: [194473800](https://github.com/Azure/oav/pull/257#discussion_r194473800) [](ancestors = 194473800)